packages/zone/src/durable.js

```diff
@@ -27,13 +26,7 @@ const { Fail } = assert;
  * @param {unknown} specimen
  * @returns {boolean}
  */
-const isStorable = specimen => {
-  try {
-    return canBeDurable(specimen);
-  } catch (_e) {
-    return false;
-  }
-};
+const isStorable = specimen => isPassable(specimen) && canBeDurable(specimen);
```

Comment:
It *should* be the case that `canBeDurable` throws only if `specimen` is not Passable. If so, this is a safe simplification. But it does indeed make `isStorable`'s no-throw guarantee more brittle, so feel free to restore the try-catch. But even then, I think you should keep the conjunction.